src/open_inwoner/components/utils.py

```diff
@@ -132,3 +135,118 @@ def render(self):
                 kwargs,
                 self.template_name,
             )
+
+
+class RenderableTag:
```

Comment:
This is a bit of a black box but the main part from InclusionTagWebTest has been used for testing for a while.

Reply:
Sorry, I don't understand why you use it. 
Can't we just use django `TemplateResponse` or `render` shortcut? It does the same things - resolves the template with it's name and then render the content. In the htmx view we return a piece of HTML, why not to do it the way django usually does it?

Reply:
Because that would require to create boilerplate template files that would only have a {% load xyz %} and the tag we want to render with a repeat of all the arguments. This class basically creates that on the fly so we need less files and repeating of argument names/values.